packages/neutrino/src/api.js

```diff
@@ -49,6 +50,8 @@ class Api {
       });
     });
 
+    options.packageJson = require(path.join(options.root, 'package.json')); // eslint-disable-line global-require
```

Comment:
The section you removed from the node preset is essentially what we need here. We need to try to require the file, but not blow up if it's not there. Maybe we should consider setting this to an empty object? Not sure...

Reply:
Meh, we can leave it undefined.